contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+### Introduction
+
+This document focuses on cStor day 2 operations workflow via the proposed specs
+of CSPC and CSP mentioned in contribute/design/cstor-operator/specs.md
+
+### Pool Expansion
+
+* “As a OpenEBS user, I should be able to add block devices to CSPC to increase
+   the pool capacity.” *
+
+Steps To Be Performed By User:
+
+1. `kubectl edit cspc your_cspc_name`
+
+2. Put the block devices in the CSPC spec against the correct kubernetes nodes.
+
+
+**Consider following example for striped pool expansion**
+
+Current CSPC is following:
+
+
+This CSPC corresponds to a striped pool on node `gke-cstor-it-default-pool-1`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-striped
+spec:
+  type: striped
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        // System generated
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+
+```
+
+Expanding striped pools -- the spec will look
+following:
+
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-striped
+spec:
+  type: striped
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        // System generated
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+        // New block device added
+      - blockDeviceName: pool-1-bd-2
+        // System generated
+        id: pool-1-bd2
+        // System generated
+        capacity: 12345234
+
+```
+
+**Consider following example for mirrored pool expansion**
+
+
+Current CSPC is following:
+This CSPC corresponds to a mirrored pool on node `gke-cstor-it-default-pool-1`
+and on node `gke-cstor-it-default-pool-2`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-mirrored
+spec:
+  type: mirrored
```

Comment:
Can we explain in comments about what is `mirrored` w.r.t this CSPC schema?

Reply:
@vishnuitta -- Can you please help me with comments e.g. comment that explain what is mirrored, striped, raidz and raidz2 , mostly with raidz and raidz2.